package.json

```diff
@@ -41,7 +41,8 @@
     "omniSharp": "1.39.11",
     "razor": "7.0.0-preview.24178.4",
     "razorOmnisharp": "7.0.0-preview.23363.1",
-    "razorTelemetry": "7.0.0-preview.24178.4"
+    "razorTelemetry": "7.0.0-preview.24178.4",
+    "xamlTools": "17.11.34815.8"
```

Comment:
If not too late, can you please change it to 17.11.34816.20. It includes first working end-to-end XAML Hot Reload.